example/lib/main.dart

```diff
@@ -96,39 +85,38 @@ class _MyAppState extends State<MyApp> {
   Widget build(BuildContext context) {
     return MaterialApp(
       home: Scaffold(
-        appBar: AppBar(
-          title: const Text('Microsoft Authentication'),
-        ),
-        body: SingleChildScrollView(
-          child: Container(
-            width: double.infinity,
-            padding: EdgeInsets.all(8),
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.start,
-              children: <Widget>[
-                RaisedButton( onPressed: _acquireTokenInteractively,
-                  child: Text('Acquire Token'),),
-                RaisedButton( onPressed: _acquireTokenSilently,
-                    child: Text('Acquire Token Silently')),
-                RaisedButton( onPressed: _signOut,
-                    child: Text('Sign Out')),
-                RaisedButton( onPressed: _fetchMicrosoftProfile,
-                    child: Text('Fetch Profile')),
-                if (Platform.isAndroid == true)
-                  RaisedButton( onPressed: _loadAccount,
-                      child: Text('Load account')),
-                SizedBox(height: 8,),
-                if (Platform.isAndroid == true)
-                  Text( "Username: $_username"),
-                SizedBox(height: 8,),
-                Text( "Profile: $_msProfile"),
-                SizedBox(height: 8,),
-                Text( "Token: $_authToken"),
-              ],
-            ),
+          appBar: AppBar(
+            title: const Text('Microsoft Authentication'),
           ),
-        )
-      ),
+          body: SingleChildScrollView(
+            child: Container(
+              width: double.infinity,
+              padding: EdgeInsets.all(8),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[
+                  RaisedButton(
+                    onPressed: _acquireTokenInteractively,
+                    child: Text('Acquire Token'),
+                  ),
+                  RaisedButton(onPressed: _acquireTokenSilently, child: Text('Acquire Token Silently')),
+                  RaisedButton(onPressed: _signOut, child: Text('Sign Out')),
+                  RaisedButton(onPressed: _fetchMicrosoftProfile, child: Text('Fetch Profile')),
+                  SizedBox(
+                    height: 8,
+                  ),
+                  SizedBox(
+                    height: 8,
+                  ),
+                  Text("Profile: $_msProfile"),
+                  SizedBox(
+                    height: 8,
```

Comment:
I'd reccomend to remove comma and format

Reply:
Formatted